pages/index.js

```diff
@@ -0,0 +1,12 @@
+import styles from "../styles/Auth.module.css"
+export default function Auth(){
```

Comment:
Please do not miss empty lines before exports and return statements if it is not a single element of a code block